lib/fastlane/plugin/wpmreleasetoolkit/models/file_reference.rb

```diff
@@ -42,7 +42,18 @@ def update
         File.write(encrypted_file_path, encrypted)
       end
 
+      # "Applies" the instruction described in the instance to the file system.
+      # That is, copies the content of the source `file` to the `destination`
+      # path.
+      #
+      # @raise [StandardError] For security reasons, it will raise if
+      # `destination` is not ignored under Git.
       def apply
+        # Only decrypt the file if the destination is ignored in Git
+        unless Fastlane::Helper::GitHelper.is_ignored?(path: destination_file_path)
+          raise StandardError.new "Attempted to decrypt #{file} to #{destination_file_path} which is not ignored under Git"
```

Comment:
Surprised that rubocop doesn't suggest you ere to directly use the String literal without `StandardError.new`, as this is the more idiomatic way iinm. That file might be excluded from rubocop though (because I never had time to finish fixing all the existing violations back then)
```suggestion
          raise "Attempted to decrypt #{file} to #{destination_file_path} which is not ignored under Git"
```

Reply:
Rubocop _did_ suggest it, once I enabled it in my Vim setup... 😅 

It also pointed out a few more violations which I'll address later in this PR, as I'm running out of time today. I'll re-request your review once done.

Thank you for pointing this out 👍